properties/handlers.py

```diff
@@ -6,10 +6,16 @@
 
 from six import string_types
 
+from .utils import everything
+
 
 def _set_listener(instance, obs):
     """Add listeners to a Properties class instance"""
-    for name in obs.names:
+    if obs.names is everything:
+        names = list(instance._props)
```

Comment:
This is wonderfully Pythonic, but I wonder if `list(instance._props.keys())` might be less magic & more clear.